tools/cargo-zerocopy/src/main.rs

```diff
@@ -197,12 +197,16 @@ fn install_toolchain_or_exit(versions: &Versions, name: &str) -> Result<(), Erro
 
 fn get_rustflags(name: &str) -> &'static str {
     if name == "nightly" {
-        "--cfg __INTERNAL_USE_ONLY_NIGHTLY_FEATURES_IN_TESTS "
+        "--cfg __ZEROCOPY_INTERNAL_USE_ONLY_NIGHTLY_FEATURES_IN_TESTS "
     } else {
         ""
     }
 }
 
+fn get_toolchain_rustflags(name: &str) -> String {
+    format!("--cfg __ZEROCOPY_TOOLCHAIN=\"{}\", name)
```

Comment:
Ooops, my original suggestion had a typo.

```suggestion
    format!("--cfg __ZEROCOPY_TOOLCHAIN=\"{}\"", name)
```